tests/TestCollectors.py

```diff
@@ -59,6 +59,8 @@ def test_collector_metrics(self):
                                                           {'name': 'resource2', 'uuid': '5628-9ba1-55e847050815'}])
         Resources.get_latest_stat = MagicMock(return_value=1)
         Resources.get_property = MagicMock(return_value="test_property")
+        Resources.get_project_ids = MagicMock(return_value=[{"3628-93a1-56e84634050814": "internal"},
```

Comment:
can we get a not internal test-case too?

Reply:
Thats not an issue.